components/InputScanner.js

```diff
@@ -36,7 +36,7 @@ class InputScanner extends React.Component {
           <Text>Camera permission is not granted</Text> :
           <BarCodeScanner
               onBarCodeRead={this._handleBarCodeRead}
-              style={{ height: 200, width: 200 }}
+              style={{ height: '100%', width: 500 }}
```

Comment:
is there not a way to make the width reactive to the screen size, rather then fixed which may cause style differences across devices